### ISSUE TITLE
SVG Icons not showing up in website

### ISSUE THREAD

**Issue by sainsb** [TheGartrellGroup/psu-campus-map#2]
SVG icons are not showing up in website.

**Comment by sainsb** [comment]
I think this has to do with the content-type header on the svg images. Not really sure how this changed from before, but I see that Apache is sending them down as text/xml when it should be sending them down as image/svg+xml.

![image](https://cloud.githubusercontent.com/assets/1097798/12678249/bcf32930-c652-11e5-8aa7-41740331f472.png)

Here is a site that describes how to affect the content-type header:
https://css-tricks.com/snippets/htaccess/serve-svg-correct-content-type/

Basically, if there isn't already, create a .htaccess file in the pdx campus map directory and place this line in it: (otherwise add this line)
`AddType image/svg+xml .svg .svgz`
